theatre/studio/src/StudioStore/createTransactionPrivateApi.ts

```diff
@@ -74,174 +74,230 @@ export default function createTransactionPrivateApi(
   stateEditors: ITransactionPrivateApi['stateEditors'],
   drafts: ITransactionPrivateApi['drafts'],
 ): ITransactionPrivateApi {
-  return {
-    set: (pointer, value) => {
-      ensureRunning()
-      const _value = cloneDeepSerializableAndPrune(value)
-      if (typeof _value === 'undefined') return
+  function setAtPosition<T>(setOptions: {
```

Comment:
defining `setAtPosition()`  is premature at this point. We don't yet have a use-case for it, and also, when we ship state-machines and multiple sequences, we'll need a different api than `setAtPosition()`. 8/10

Fine with renaming `set()` to `setAtPlayhead()` for clarity. 2/10

Reply:
Applied